src/services/near-nep21-util.js

```diff
@@ -121,6 +121,23 @@ export async function gasCheck() {
   return false;
 }
 
+// Use decimal attribute format and return token amount.
+// Till 5 decimals
+export function convertToDecimals(str, decimals) {
+  let len = Number(decimals);
+  if(len == 0) {
+    return str;
+  }
+
+  let result = (str + "").padStart(len + 1, "0");
```

Comment:
why we do `str + ""`?

Reply:
this is used to convert `str` to string type. Updating to `String(str)`. this seems more clear